src/apify/consts.py

```diff
@@ -1,208 +1,5 @@
 import re
 from enum import Enum
```

Comment:
We'll have to do the reexport magic here, same as in client, so it's not a breaking change